qhbmlib/utils.py

```diff
@@ -90,3 +90,41 @@ def expand_unique_results(y, idx):
   """
   expanded = tf.gather(y, idx, axis=0)
   return expanded
+
+
+def bitstrings_to_integers(bitstrings):
```

Comment:
can we move these fns to a separate draft PR since they're unused (IIUC) for now?

Reply:
(the reason is because only introducing functions which are currently serving a functional purpose enhances readability. if we have any existing fns that are unused then we may want to remove them in a separate PR. it's easy to rollback when they become necessary again.)